src/tests/ftest/util/test_utils.py

```diff
@@ -247,6 +352,39 @@ def destroy(self, force=1):
             return True
         return False
 
+    @fail_on(DaosApiError)
+    def destroy_dmg(self, dmg_bin_path, force=1):
+        """Destroy the pool using dmg.
+
+        Args:
+            dmg_bin_path (str): Directory where dmg is installed. Call
+                self.basepath + '/install/bin' in the test
+            force (int, optional): force flag. Defaults to 1.
+
+        Returns:
+            bool: True if the pool has been destroyed; False if the pool is not
+                defined.
+        """
+        if self.pool:
+            self.disconnect()
+            self.log.info("Destroying pool %s", self.uuid)
+            if self.pool.attached:
+                if force == 1:
+                    force_bool = True
+                else:
+                    force_bool = False
+                destroy_result = pool_destroy(path=dmg_bin_path,
+                                              pool_uuid=self.uuid,
+                                              force=force_bool)
+                self.log.info(" Destroy result stdout = %s" %
```

Comment:
(pylint-logging-not-lazy) Specify string format arguments as logging function parameters